### PR TITLE
Marketingauto kbr

### PR DESCRIPTION
**Pull request by kbr-odoo**
to be merged (:

### CODE REVIEW

File: marketing_automation.rst [hunk 1]
```diff
@@ -0,0 +1,13 @@
+:banner: banners/marketing_automation.jpg
```
Comment (AntoineVDV) [inline marketing_automation.rst]:
```suggestion
:banner: banners/marketing_automation.png
```

File: marketing_automation/automate_actions.rst [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+
+
+.. image:: media/auto1.png
+   :align: center
+   :alt: Create workflows in Odoo Marketing Automation
+
+
+Depending on the criteria, the *target model updates itself* while the campaign is running. That
+means that every new record that meets the criteria will be added to the workflow.
+Example: a campaign starts for leads that have no salesman assigned. If at some point, those leads
```
Comment (AntoineVDV) [inline marketing_automation/automate_actions.rst]: Line break intended here ?

File: marketing_automation/automate_actions.rst [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+amount of time and in subsequence to the trigger.
+
+Under *Domain* you can narrow down your target, even more, applying filters for this specific
+activity.
+
+
+Add Child Activities
+--------------------
+A *child activity* is an action that will take place based on a condition.
+You can either create or add a child activity if on the previous one: the email was opened or not,
```
Comment (AntoineVDV) [inline marketing_automation/automate_actions.rst]: Line break intended here ?

File: marketing_automation/automate_actions.rst [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+the email was replied or not, the link was clicked or not, the email bounced back.
+
+
+.. image:: media/auto3.png
+   :align: center
+   :alt: Create workflows in Odoo Marketing Automation
+
+.. seealso::
+   :doc:`../marketing_automation/segment`
```
Comment (AntoineVDV) [inline marketing_automation/automate_actions.rst]:
```suggestion
   :doc:`segment`
```
`../marketing_automation/` means go to the parent folder of marketing_automation and then back into the marketing_automation folder to look for the page segment.

File: marketing_automation/segment.rst [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+   :alt: Filters in Odoo Marketing Automation
+
+
+Every time a new branch is created, you can choose if you would like the records to match *all*
+criteria of the upcoming rules, or *any*, meaning one rule or the other.
+
+
+Use cases
+=========
+**Scenario**: narrow the target down to new opportunities in the pipeline.
```
Comment (AntoineVDV) [inline marketing_automation/segment.rst]: Content nitpicking: I'd put each scenario in a separate bullet point with the image below aligned left to have a clearer split of the various use cases.

File: marketing_automation/segment.rst [hunk 6]
```diff
@@ -0,0 +1,81 @@
...
+
+**Scenario**: narrow the target down to records with a helpdesk ticket solved.
+
+
+.. image:: media/filters8.png
+   :align: center
+   :alt: Filters in Odoo Marketing Automation
+
+.. seealso::
+   :doc:`../marketing_automation/automate_actions`
```
Comment (AntoineVDV) [inline marketing_automation/segment.rst]:
```suggestion
   - :doc:`automate_actions`
```

File: marketing_automation/test.rst [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+
+
+.. image:: media/test3.png
+   :align: center
+   :alt: Create workflows in Odoo Marketing Automation
+
+
+If you click on *Participants* and open a record, you will find the specific activities the record
+went/are/will go through, as well as when the workflow was triggered to him.
+If you wish to cancel an activity click on *Cancel*.
```
Comment (AntoineVDV) [inline marketing_automation/test.rst]: Did you intend a line break here ?

File: marketing_automation/test.rst [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+
+.. seealso::
+   :doc:`../marketing_automation/automate_actions`
```
Comment (AntoineVDV) [inline marketing_automation/test.rst]:
```suggestion
   - :doc:`automate_actions`
```

File: marketing_automation/test.rst [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
```
Comment (AntoineVDV) [inline marketing_automation/test.rst]:
```suggestion
```
Technical nitpicking: Empty line not needed here

File: marketing_automation/test.rst [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+   :doc:`../marketing_automation/segment`
```
Comment (AntoineVDV) [inline marketing_automation/test.rst]:
```suggestion
   - :doc:`segment`
```

File: marketing_automation/understanding_metrics.rst [hunk 11]
```diff
@@ -0,0 +1,88 @@
+=====================
+Understanding Metrics
+=====================
+Metrics are values that help you measure progress and can be a powerful way of linking your
+employees to goals.
+
+
+When you create a workflow in Odoo, its visual content already shows metrics in a graph form and
+in numbers.
+Let’s consider the example below:
```
Comment (AntoineVDV) [inline marketing_automation/understanding_metrics.rst]: Line break ?

File: marketing_automation/understanding_metrics.rst [hunk 12]
```diff
@@ -0,0 +1,88 @@
...
+
+
+.. image:: media/metrics1.png
+   :align: center
+   :alt: Metrics in Odoo Marketing Automation
+
+
+The *Target* - business object - is *Lead/Opportunity* and was narrowed down to the ones whose
+*Tag Name* contain the description “Product”, and have an email address set.
+A total number of 20 records match the criteria.
```
Comment (AntoineVDV) [inline marketing_automation/understanding_metrics.rst]: Line break ?

File: marketing_automation/understanding_metrics.rst [hunk 13]
```diff
@@ -0,0 +1,88 @@
...
+.. note::
+   Every time a new record is added to the *Target* model, it will be automatically added to the
+   workflow, and, it will start the workflow from the beginning (parent action).
+
+.. seealso::
+   :doc:`../marketing_automation/automate_actions`
+
+   :doc:`../marketing_automation/segment`
+
+   :doc:`../marketing_automation/test`
```
Comment (AntoineVDV) [inline marketing_automation/understanding_metrics.rst]: Same as above.

**Comment by kbr-odoo** [comment]
Do you think you could merge this one and email marketing, please?
thanks.

**Comment by AntoineVDV** [comment]
@kbr-odoo @jubodoo has to merge them, I just made a tech review ;)

**Pull request merged**